packages/react-button/src/components/Button/Converged.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
```

Comment:
I'm not sure we need to worry too much about that right now. Like I mentioned in the previous reply, these files are just copies, so not a change to the status quo. More generally, there's nothing storybook-specific about this format aside from the name (it's just a file exporting a bunch of components), so moving to some other format shouldn't be too hard.

Reply:
>  I would really rather not have two separate systems for code examples.

yeah I fully agree. I just copied button stories from react-examples to be collocated. Ideally react-examples shouldn't exist, or if it does only as an e2e suite aggregator.

> how to avoid lock in with storybook stories

Heh but we are locked in already with webpack/jest/react/<name your tool here> ...

What's important is to have solid universal format for "real" docs ( md/mdx ). With that format/metadata we can switch to anything basically (what @ecraig12345  mentioned). 

_out off topic:_ from what I heard the plan is to have 1 storybook to aggregate all components out there from various repos

> so not a change to the status quo.

From my experience, consistency should be primary goal (simplicity,Dx...). With aforementioned setup, we are not consistent - eg. we have test collocated with implementation but docs live somewhere completely different . TBH I don't see any benefit in that.

Reply:
> From my experience, consistency should be primary goal (simplicity,Dx...). With aforementioned setup, we are not consistent - eg. we have test collocated with implementation but docs live somewhere completely different . TBH I don't see any benefit in that.

The only way we could have docs live in the same package is if we can find a way around the circular dependency issues mentioned elsewhere--which would be nice but seems unlikely.